### ISSUE TITLE
`fetch_all_slab_keys` and `fetch_stats` do not refresh

### ISSUE THREAD

**Issue by rubyconvict** [bananastalktome/memcached_stats#4]
```
stats = MemcachedStats.new
stats.fetch_all_slab_keys
stats.slab_keys
stats.fetch_all_slab_keys
stats.slab_keys # new keys not listed
stats.fetch_stats # new keys not listed
```
When I instantiate again it's ok:
```
stats = MemcachedStats.new
stats.fetch_all_slab_keys
stats.slab_keys # all keys listed
stats.fetch_stats # all keys listed
```
I'm not sure if it's a bug or feature. Is there any other way to get fresh output?

**Comment by bananastalktome** [comment]
I pushed a small fix (version 0.0.5) which might fix this issue - I don't shortcut the `get_slabs` call within `fetch_all_slab_keys` if it has already been run. Please let me know if this fixes things for you.

**Comment by rubyconvict** [comment]
Yes, thank you.
